db/CSV_Writers/csvMachine.js

```diff
@@ -0,0 +1,17 @@
+const {
+  logs: { log, logStart, logVictoryMessage },
+  utils: { drainWriter, postDBQuery }
+} = require('./config/config.js');
+
+module.exports = async function ({ title } = this) {
```

Comment:
i saw this syntax in your last PR and was unsure how it worked. it looks like destructuring. are you accessing the title value of whatever object you pass as an argument?

Reply:
_this_ isn't passed as an argument, if it were I could just write { title } inside of the parameter list. 
I'm using `call` and destructuring inside of the parameter list to save exactly one line of code from being written.

Otherwise it would look like

function(provision) {
  const {title} = provision //this is the line!
  ...
};

I made changed it back, way too trivial.